### ISSUE TITLE
[css-grid]min-content/max-content prefix in Safari

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#803]
input:

```css
.foo {
     grid-template-columns: minmax(100px, min-content);
}
```

output:

```css
.foo {
     -ms-grid-columns: minmax(100px, min-content);
         grid-template-columns: minmax(100px, min-content);
}
```

These properties just remove the prefix in https://trac.webkit.org/changeset/213831 , but I'm not sure which version of Safari.

@mrego Can you help me?

http://caniuse.com/#feat=css-grid

**Comment by mrego** [comment]
Apple doesn't publicly talk about their Safari releases, so we don't know.
I don't think it would be part of 10.1 as the beta has been around for a while, so probably Safari 11 or 10.2 if that ever exists.
Sorry for not being really helpful, you'd need to wait and check when it's or not included.

**Comment by ai** [comment]
@yisibl what is expected output?

**Comment by yisibl** [comment]
@mrego Well, it looks like caniuse's data is inaccurate.

@ai Maybe we don't need a prefix in `minmax(100px, -webkit-min-content)`,We have to wait for Safari to officially support Grid.

**Comment by ai** [comment]
@yisibl ouh, I will add grid properties to `min-content` https://github.com/postcss/autoprefixer/blob/master/data/prefixes.coffee#L292

Do you want to make PR to save your name in history (again)? :)

**Comment by mrego** [comment]
One special thing here, `fit-content` cannot be prefixed inside grid properties, or it won't work. I know it's a weird situation but that's how things are right now.
So valid declarations would be:
```grid-template-columns: -webkit-min-content -webkit-max-content fit-content(500px);```
However the following one would be invalid:
```grid-template-columns: -webkit-min-content -webkit-max-content -webkit-fit-content(500px);```

Note that `fit-content` in the grid properties is not a keyword like `min|max-content` but a kind of "function". That's the reason why it was implemented as something new and unprefixed since the beginning.

**Comment by yisibl** [comment]
@mrego OK, I see, `fit-content()` no prefix.

@ai  Yeah, willing to serve you 👻 .

**Comment by yisibl** [comment]
Safari 10.1 shipped: https://developer.apple.com/library/prerelease/content/releasenotes/General/WhatsNewInSafari/Articles/Safari_10_1.html#//apple_ref/doc/uid/TP40014305-CH12-DontLinkElementID_55

**Comment by yisibl** [comment]
Add test case: https://github.com/postcss/autoprefixer/pull/805

**Comment by ai** [comment]
Issue with test pull request — ideal issue :+1: 

Thanks, it will help me a lot.

**Comment by ai** [comment]
Done 99def37

**Comment by ai** [comment]
Released in 7.1

**Comment by jayenashar** [comment]
Hi I've got a related issue where:

    grid-template: max-content 1fr max-content max-content / max-content 1fr;

is getting prefixed as:

    grid-template: -webkit-max-content 1fr -webkit-max-content max-content / -webkit-max-content 1fr;

I expect that to be:

    grid-template: -webkit-max-content 1fr -webkit-max-content -webkit-max-content / -webkit-max-content 1fr;

that is, `-webkit-` is missing from the third `max-content`, but not the first, second, or fourth...

**Comment by ai** [comment]
@jayenashar confirmed. I will try to fix it on next week.

**Comment by jayenashar** [comment]
that was FAST

**Comment by ai** [comment]
@jayenashar fixed ecd7eed

**Comment by ai** [comment]
@jayenashar released in 7.2.5.

**Comment by jayenashar** [comment]
thanks!
